### PR TITLE
Feature/chips

### PR DESCRIPTION
**Pull request by gwelter**
Added Chip component

## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: src/stories/Chip.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- Chip.stories.mdx -->
+
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Chip } from './../components/Chip';
+
+---
+name: Sonr Chip
+route: /Components/Chip
```
Comment (joshLong145) [inline src/stories/Chip.stories.mdx:8]: this should match the rest of the stories.

File: src/components/Chip/Chip.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const cx = require('classnames');
+
+export const chipsSkins: { [key in ChipSkin]: string } = {
+  default: 'bg-skin-subtle-inverted bg-opacity-10 text-skin-subtle-inverted',
+  primary: 'bg-blue-700 text-white',
+  critical: 'bg-red-700 text-white',
+}
+
+export const Chip: React.FC<ChipProps> = ({ skin = 'default', text, onRemove }) => {
+  const chipClasses = useMemo(() => cx('flex items-center rounded w-fit px-2 py-1.5', chipsSkins[skin]), [skin]);
```
Comment (joshLong145) [inline src/components/Chip/Chip.tsx:14]: using a flex container might cause alignment problems when used with an outer layout controller. didn't we refactor the other controls to no longer use this?

**Review by joshLong145 — CHANGES_REQUESTED** [review]

File: src/components/Chip/Chip.tsx (hunk 2 above)
Comment (gwelter) [reply src/components/Chip/Chip.tsx:14]: Hi, i don't remember we refactoring that actually. But i can check that.

File: src/stories/Chip.stories.mdx (hunk 1 above)
Comment (gwelter) [reply src/stories/Chip.stories.mdx]: Yep, just fixed that.

File: src/components/Chip/Chip.tsx (hunk 2 above)
Comment (joshLong145) [reply src/components/Chip/Chip.tsx:14]: looking back this what not what i was thinking of.

**Review by joshLong145 — APPROVED** [review]

File: src/components/Chip/Chip.tsx (hunk 2 above)
Comment (gwelter) [reply src/components/Chip/Chip.tsx]: But it worked so i've removed it.

**Pull request merged**